README.md

```diff
@@ -1,9 +1,28 @@
-# semantic-release-error
+# **@semantic-release/error**
```

Comment:
I don’t think we need the `**` bold formatting when it’s already a level 1 heading

Reply:
Removed